### DISCUSSION TITLE
[Feature]: 该评论仅登录用户或者评论双方可见

### DISCUSSION THREAD

**Discussion by appotry** [walinejs/waline#1514] [Ideas]
### 功能概述 | Describe the feature

有时候，想发表评论，但是仅仅只想给特定的对象可见，而不想公开发布出来

这时候，这个功能就用得着了

**Comment by appotry** [comment]
一种使用场景，一个下载链接放到评论区，只给登录用户可见

如果可以实现评论的特定部分登录可见就更好了

比如

```
下载链接 云盘 链接 https://XXXXXXX

[加密]提取码 [/加密]
```

**Reply by lizheming** [reply]
你可以调用 Waline API 自行实现该功能，不建议把这种特殊逻辑放到评论列表里面来。

**Reply by appotry** [reply]
相关的api是哪一个？

个人感觉，这个功能是增加人气非常好的一个手段

**Reply by lizheming** [reply]
https://waline.js.org/api/#api-User-GetToken 需要带上 `authorization: Bearer xxx` 的头。
